mne/simulation/raw.py

```diff
@@ -0,0 +1,614 @@
+# Authors: Mark Wronkiewicz <wronk.mark@gmail.com>
+#          Yousra Bekhti <yousra.bekhti@gmail.com>
+#          Eric Larson <>
+#
+# License: BSD (3-clause)
+
+import os
+from os import path as op
+import numpy as np
+import warnings
+from copy import deepcopy
+
+from mne import (pick_types, pick_info, pick_channels, VolSourceEstimate,
+                 convert_forward_solution, get_chpi_positions, EvokedArray,
+                 make_ad_hoc_cov)
+from mne.bem import fit_sphere_to_headshape, make_sphere_model
+from mne.io import read_info, Raw
+from mne.externals.six import string_types
+from mne.forward.forward import _merge_meg_eeg_fwds, _stc_src_sel
+from mne.forward._make_forward import (_prep_channels, _setup_bem,
+                                       _compute_forwards, _to_forward_dict)
+from mne.forward._compute_forward import _magnetic_dipole_field_vec
+from mne.transforms import _get_mri_head_t, transform_surface_to
+from mne.source_space import (SourceSpaces, read_source_spaces,
+                              _filter_source_spaces, _points_outside_surface)
+from mne.bem import _bem_find_surface
+from mne.io.constants import FIFF
+from mne.io.meas_info import Info
+from mne.source_estimate import _BaseSourceEstimate
+from mne.utils import logger, verbose, check_random_state
+from mne.simulation import generate_noise_evoked
+
+
+@verbose
+def simulate_raw(raw, stc, trans, src, bem, cov='simple', pos=None, ecg=False,
+                 blink=False, chpi=False, mindist=1.0, interp='linear',
+                 random_state=None, n_jobs=1, verbose=None):
+    """Simulate raw data with head movements
+
+    Parameters
+    ----------
+    raw : instance of Raw
+        The raw instance to use. The measurement info, including the
+        head positions, will be used to simulate data.
```

Comment:
@agramfort brought up a good point that it would be nice to not have to pass a raw. We were thinking of just using an `info` object, but there are other dependencies (`raw.times`, `raw.tstep`, `raw.first_samp`, etc.) Can we get all that from stcs?